File: problemas/iniciante/1589/1589.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
    
        int T = Integer.parseInt(in.readLine());
        for(int i = 0; i < T; ++i){
            String[] entrada = in.readLine().trim().split(" ");
            System.out.println(Integer.parseInt(entrada[0]) + Integer.parseInt(entrada[1]));
        }
    }
}

File: problemas/iniciante/1589/1589.exs
defmodule Modulo do
  def leitura(n) do
    if n > 0 do
      entrada = IO.gets("") |> String.trim |> String.split(" ")

      r1 = entrada |> Enum.at(0) |> String.to_integer()
      r2 = entrada |> Enum.at(1) |> String.to_integer()

      IO.puts (r1 + r2)
      
      leitura(n - 1)
    end
  end
end

n = IO.gets("") |> String.trim |> String.to_integer()
Modulo.leitura(n)